custom-recipes/nlp-analysis-ontology-tagging/recipe.json

```diff
@@ -369,8 +369,18 @@
             "label": "Lemmatize",
             "description": "Simplify words to their lemma form e.g. going → go, mice → mouse",
             "type": "BOOLEAN",
+            "visibilityCondition": "model.language != 'language_column' && ['es', 'nb', 'fr', 'de', 'ru', 'pl','ca','cs','da','hr','hu','id','it','lb','lt', 'pt', 'ro','sr','tl','tr', 'ur', 'en','mk','nl','sv','bn','el','fa'].includes(model.language)",
             "defaultValue": false
         },
+        {
+            "name": "lemmatization_multilingual",
+            "label": "Lemmatize",
+            "description": "Simplify words to their lemma form e.g. going → go, mice → mouse. Available for 28 out of 59 languages.",
+            "type": "BOOLEAN",
+            "visibilityCondition": "model.language == 'language_column'",
```

Comment:
If I understand correctly, depending on the language setting only one of the two lemmatizers will by shown

Reply:
Exactly! Here is an explanation about it : 
We can support only 28 of the 59 supported languages for lemmatization.
-In case the user wants to run the recipe with only one given language: we display this lemmatizer button https://github.com/dataiku/dss-plugin-nlp-analysis/pull/18#discussion_r630196249 only if the choosen language is suported
-In case the user wants to run the recipe for multiple languages, he provides us a language column (e.g model.language==language_column), and we will display the other lemmatizer button and the recipe will raise an error if a language from language_column is unsupported